test/ae_mdw/db/state_test.exs

```diff
@@ -127,59 +127,81 @@ defmodule AeMdw.Db.StateTest do
   end
 
   describe "commit_mem" do
-    test "saves aex9 state into ets store" do
-      ct_pk = :crypto.strong_rand_bytes(32)
+    test "saves multiple aex9 task results into ets store" do
       block_index = {567_891, 2}
       call_txi = 12_345_680
 
       account_pk = :crypto.strong_rand_bytes(32)
       amount = Enum.random(1_000_000_000..9_999_999_999)
       balances = %{{:address, account_pk} => amount}
 
-      Aex9BalancesCache.put(ct_pk, block_index, @next_hash, balances)
+      ct_pks =
+        Enum.map(1..100, fn _i ->
```

Comment:
Locally with 16 threads can process 30x more within same time. There will be a PR to optimize it by not calculating the hash within the task.